nemo_curator/scripts/semdedup/clustering.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+from datetime import datetime
+
+os.environ["DASK_DATAFRAME__QUERY_PLANNING"] = "False"
```

Comment:
Can this be fixed instead by moving the `dask_cudf` import to after the `nemo_curator` imports, or will the style conformation not let you?

Reply:
yeah, the style conformation does not let me.  That was my first attempt to fix it.